TechAndCheck/zenodotus
# Add usability to status page

This adds functionality to automatically update the jobs status page in
real time. It does this by adding a new ActionCable channel to monitor
scrape objects, as well as moving everything over to Stimulus styling.

This adds a few new routes to add Turbo functionality to make everything
*click* together. It also adds basic pagination.

To test:
- Start server using `./bin/dev`
- Start up Redis and Sidekiq
- Go to job status page
- Open new tab in app and archive a url using the manual process
- Go back to the status page and you should see a new "pill" appear
  showing new number of scrapes
- click on the pill, the table should reload
- Click next and previous to scroll through pages

Closes https://github.com/TechAndCheck/zenodotus/issues/288
